llmfoundry/data/finetuning/tasks.py

```diff
@@ -176,8 +176,6 @@ def __init__(self,
             shuffle_seed=shuffle_seed,
             shuffle_block_size=shuffle_block_size,
             sampling_method=sampling_method,
-            sampling_granularity=sampling_granularity,
```

Comment:
Both are valid arguments based on streaming 0.6.0 release: [link](https://github.com/mosaicml/streaming/blob/v0.6.0/streaming/base/dataset.py#L279). Are you using older streaming version ?